src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -611,6 +621,22 @@ private ObjectType getSuperType(FunctionType type) {
     return "Object".equals(implicitProto.getDisplayName()) ? null : implicitProto;
   }
 
+  private String getUnqualifiedName(TypedVar symbol) {
+    return getUnqualifiedName(symbol.getName());
+  }
+
+  private String getUnqualifiedName(String name) {
+    return lastPart(name);
+  }
+
+  private String lastPart(String input) {
```

Comment:
Is there a better name for this?  E.g. lastDottedComponent or something?  I don't like my name much either but lastPart is so vague.